color_refinement.py

```diff
@@ -4,12 +4,28 @@
 """
 from color_refiment_helper import *
 import time
+import preprocessing
 from graph_io import *
 
 PATH = './graphs/treepaths/'
 GRAPH = 'threepaths160.gr'
 
 
+def preprocess_graphs(g: "Graph", h: "Graph") -> bool:
```

Comment:
Maybe give it another name, this suggests that the graph is altered. However it checks if g and h are candidates for being isomorphic